### PR TITLE
refactor-sikuli-run-interface

### PR DESCRIPTION
**Pull request by ShakoHo**
@ypwalter r?

### CODE REVIEW

File: lib/perfBaseTest.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def tearDown(self):
             desktopHelper.stop_browser(self.browser_type, self.env)
 
         # Delete Url
-        if self.test_url_id:
+        if hasattr(self,"test_url_id"):
```
Comment (ypwalter) [inline lib/perfBaseTest.py]: Is there any situation that we might not have test_url_id?

File: suite.txt [hunk 2]
```diff
@@ -1 +1 @@
-test_chrome_gdoc_pd3
+test_firefox_gdoc_read_basic_table_1
```
Comment (ypwalter) [inline suite.txt]: XD?

File: lib/perfBaseTest.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ def setUp(self):
         self.target_helper = targetHelper.TagetHelper(self.env)
 
         # init sikuli
-        self.sikuli = sikuli.Sikuli()
+        self.sikuli = sikuli.Sikuli(self.env.run_sikulix_cmd_path, self.env.hasal_dir)
```
Comment (ypwalter) [inline lib/perfBaseTest.py]: I didn't see you commit Environment.py in. There is no such default variable currently.

File: lib/perfBaseTest.py (hunk 1 above)
Comment (ShakoHo) [inline lib/perfBaseTest.py]: yes, when you use pilot testing (given --sikuli_script when executing runtest.py), there will be no test_url_id assign in test script

File: suite.txt (hunk 2 above)
Comment (ShakoHo) [inline suite.txt]: XD...no meaning change sorry..

**Comment by ypwalter** [comment]
r+ as it is running smooth in windows now.

**Pull request merged**